### DISCUSSION TITLE
How do I create a simulator .app file using fastlane?

### DISCUSSION THREAD

**Discussion by alexvbush** [fastlane/fastlane#19933] [Q&A]
hi, I have the following lane that produces a QA build for Firebase distribution:
```ruby
lane :build_development do
    staging_certificates
    build_app(workspace: "upkeep.xcworkspace", 
              scheme: "upkeep",
              configuration: "Debug",
              export_method: "development")
  end
```

How can I make a `.app` file instead that I can use in my smoke tests?

**Comment by tahirmt** [comment]
We just copy the `.app` file located under your derived data `<Derived data path>/Build/Products/Debug-iphonesimulator` location and use it for automation tests.
